float64 volume
float64 mix_left
float64 mix_right
string file_path
